ee/indexeddb/indexeddb.go

```diff
@@ -64,7 +67,11 @@ func QueryIndexeddbObjectStore(dbLocation string, dbName string, objectStoreName
 	}
 	databaseIdRaw, err := db.Get(databaseNameKey, nil)
 	if err != nil {
-		return nil, fmt.Errorf("querying for database id: %w", err)
+		// If the database doesn't exist, return an empty list of objects
+		if errors.Is(err, leveldberrors.ErrNotFound) {
+			return objs, nil
+		}
+		return nil, fmt.Errorf("querying for database id: %w", err) // TODO if leveldb: not found, return no rows?
```

Comment:
not sure i understand the TODO comment here but probably not important

Reply:
lol whoops, that was an implementation note I'd left for myself. removed, thank you!